packages/generator/src/edmx-to-vdm/common/action-function-parameters.ts

```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import { VdmParameter } from '../../vdm-types';
+import { filterUnknownEdmTypes, parseType } from '../edmx-to-vdm-util';
+import { edmToTsType, isNullableParameter } from '../../generator-utils';
+import { parameterDescription } from '../description-util';
+import { EdmxNamed, EdmxParameter } from '../../edmx-parser/common';
+import { SwaggerPath } from '../../swagger-parser/swagger-types';
+import { ServiceNameFormatter } from '../../service-name-formatter';
+import { EdmxFunctionImport as EdmxFunctionImportV2 } from '../../edmx-parser/v2';
+import {
+  EdmxActionImport,
+  EdmxFunctionImport as EdmxFunctionImportV4
+} from '../../edmx-parser/v4';
+
+export function getActionImportParameters(
+  edmxActionImport: EdmxActionImport,
+  edmxParameters: EdmxParameter[],
+  swaggerDefinition: SwaggerPath | undefined,
+  formatter: ServiceNameFormatter
+): VdmParameter[] {
+  return getParameter(
+    edmxActionImport,
+    edmxParameters,
+    swaggerDefinition,
+    formatter
+  );
+}
+
+export function getFunctionImportParameters(
```

Comment:
As those two functions are basically the same, why don't we just export `getParameters`?